app/src/main/java/fmuntenescu/playground/vectordrawables/DrawListener.java

```diff
@@ -0,0 +1,14 @@
+package fmuntenescu.playground.vectordrawables;
+
+/**
+ * Listener that notifies when a view has been redrawn
+ */
+public interface DrawListener {
+
+    /**
+     * Called when the view has been redrawn
+     *
+     * @param miliseconds the time took to draw the view.
+     */
+    void viewDrawn(long miliseconds);
```

Comment:
how about 
ViewRedrawnListener with onRedraw()
and then you can remove all the javadoc